src/charm.py

```diff
@@ -187,8 +192,16 @@ def _on_install(self, _: InstallEvent) -> None:
 
     def check_required_relations(self) -> None:
         """Run check if any required relations are missing"""
-        if self.model.get_relation(self.CEPH_RELATION) is None:
-            self.unit.status = BlockedStatus("Missing relations: ceph")
+        missing_relations = []
+        if self.model.get_relation(self.CEPH_ADMIN_RELATION) is None:
```

Comment:
Not sure if there's such case but I do it like this to avoid that possibility :D
[The docs](https://ops.readthedocs.io/en/latest/#ops.model.Model.get_relation) mention that `None` means no relation so I with it.

Reply:
Ok, that's good point. I'll try to use it also :+1: